airbyte-integrations/connectors/destination-postgres/src/main/kotlin/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresSqlGenerator.kt

```diff
@@ -207,15 +207,19 @@ class PostgresSqlGenerator(
                         DSL.`val`(null as String?)
                     )
                     .else_(DSL.cast(field, SQLDataType.VARCHAR))
+            var cleanedText = extractAsText
+            if (type == AirbyteProtocolType.NUMBER) {
```

Comment:
nit: can this also handle INTEGER?

Reply:
There's no issue with integer. and as far as I can tell it's specific to numeric columns`AirbyteProtocolType.NUMBER -> "numeric"`